persistence/sql/README.md

```diff
@@ -0,0 +1,35 @@
+# SQL Package
+
+The SQL package is a library of functions that all take a `pgx.Tx` as an argument and execute SQL queries on it.
+
+- GetAccountsUpdated
+- GetActors
+- GetTransactions
+- GetPools
+- GetAccounts
+- GetFlags
+- GetParams
+
+These functions are meant to abstract away the SQL row and error handling for components of the persistence package.
+
+GetActors is used to get each of the actor types in the system: Applications, Validators, Watchers, and Servicers.
+
+## Why a whole package?
+
+If the SQL handling lives in the persistence package, the package submodules will run into import cycle errors when attempting to use the SQL handlers. Putting the SQL helpers into a package inside persistence avoids the import cycle.
+
+```mermaid
+flowchart TD
+    subgraph pers[persistence]
+        subgraph p[ without sql package - import cycle error ]
+            persistenceContext1[PersistenceContext]
+            treeStore1[TreeStore]
+            treeStore1 -- Tx --> persistenceContext1
+            persistenceContext1 -- Tx --> treeStore1
+        end 
+        subgraph p2[ with sql package - refactored ]
+            persistenceContext -- Tx --> sql
+            treeStore2 -- Tx --> sql
+        end 
+    end
+```
```

Comment:
@Olshansk the import cycle issue is described here.

Reply:
![Screenshot 2023-06-16 at 4 00 36 PM](https://github.com/pokt-network/pocket/assets/1892194/832bff0b-c229-4475-96b3-86ae368adb6e)

I'm in full support of this :)